images/base/Dockerfile

```diff
@@ -92,6 +93,23 @@ RUN wget --no-check-certificate --quiet -O /tmp/pyenv-installer https://raw.gith
     && ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/${PYTHON_VERSION}/lib/python${PYTHON_VERSION%.*}/site-packages/lsb_release.py \
     && pip install -q --upgrade --no-cache-dir pip
 
+# Python +3.10 requires OpenSSL 1.1.x, but Ubuntu 16.04 LTS ships with OpenSSL 1.0.x
+RUN wget --no-check-certificate --quiet -O /tmp/openssl-1.1.1w.tar.gz https://github.com/openssl/openssl/releases/download/OpenSSL_1_1_1w/openssl-1.1.1w.tar.gz \
```

Comment:
does it work with a recent OpenSSL 3.x? as 1.x is already EOL

Reply:
Good question! Let me try with OpenSSL 3.x. Thank you for reviewing!